src/ServiceControl/MessageFailures/Api/ArchiveMessagesController.cs

```diff
@@ -2,17 +2,14 @@ namespace ServiceControl.MessageFailures.Api
 {
     using System.Collections.Generic;
     using System.Linq;
-    using System.Net;
-    using System.Net.Http;
     using System.Threading.Tasks;
-    using System.Web.Http;
     using InternalMessages;
     using NServiceBus;
-    using Infrastructure.WebApi;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Net.Http.Headers;
     using ServiceControl.Persistence;
-    using ServiceControl.Persistence.Infrastructure;
 
-    class ArchiveMessagesController : ApiController
+    class ArchiveMessagesController : Controller
```

Comment:
Maybe inherit from ControllerBase and [ApiController] attribute. Controller adds things like ViewBag and ViewResult which are unnecessary for API endpoints. [[ApiController](https://learn.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-8.0#apicontroller-attribute)] adds a bunch of nice framework goodies that can be useful with APIs.  (The attribute can be added at the method, class or assembly level. (copy paste this comment for other controllers.